addons/account/tests/test_account_move_reconcile.py

```diff
@@ -2638,9 +2687,10 @@ def test_reconcile_cash_basis_workflow_multi_currency(self):
             {'debit': 0.0,      'credit': 0.0,      'amount_currency': -0.001,  'currency_id': currency_id,     'account_id': self.tax_account_2.id},
         ])
 
+
         self.assertAmountsGroupByAccount([
             # Account                               Balance     Amount Currency
-            (self.cash_basis_transfer_account,      -5.54,      -22.226),
+            (self.cash_basis_transfer_account,      -7.41,      -22.226),
```

Comment:
I'm not sure I understand the diff
* Company: `-11.11 + 4.17 + 0.0 + 1.4 + 0.0 = -5.54`
* Foreign: `-33.34 + 8.333 + 0.003 + 2.777 + 0.001 = -22.226`

Or are there missing lines in the `assertRecordValues` above?

Reply:
It's probably the exchange difference because it's there to ensure to rate is always satisfied on residual amounts too. In that case, 22.226 / 7.41 = 2.999460189 ~= 3 (your currency rate)

Reply:
Yes, my point is that it would be nice to see where those lines are, and on which accounts are the counterparts

Reply:
I'll add an assert :)